packages/oui-tile/_mixins.less

```diff
@@ -0,0 +1,148 @@
+@import './_variables';
+
+#oui {
+  .tile-base(
+    @tile-selector,
+    @dropdown-menu-selector,
+    @icon-selector
+  ) {
+    padding: @oui-tile-padding;
+    background-color: @oui-tile-background-color;
+    border: 1px solid @oui-tile-border-color;
+    border-radius: @oui-tile-border-radius;
+    box-shadow: @oui-tile-box-shadow;
+
+    &__title {
+      margin-left: @oui-tile-item-side-spacing;
+      margin-right: @oui-tile-item-side-spacing;
+      font-weight: 600;
+      color: @oui-tile-title-color;
+    }
+
+    &__button {
+      margin: 0;
+      padding-left: @oui-tile-item-side-spacing;
+      padding-right: @oui-tile-item-side-spacing;
+      font-weight: 600;
+      text-align: left;
+      text-decoration: none;
+
+      &:disabled {
+        text-decoration: none;
+      }
+
+      .@{icon-selector} {
+        color: @oui-tile-button-color;
+
+        &::before {
+          font-size: rem-calc(12);
+        }
+      }
+    }
+
+    &__item {
+      position: relative;
+
+      &::before {
+        content: "";
+        display: block;
+        margin: 0 @oui-tile-item-side-spacing;
+        height: 1px;
+        background-color: @oui-tile-item-border-color;
+      }
+
+      &:last-child {
+        &::after {
+          content: "";
+          display: block;
+          margin: 0 @oui-tile-item-side-spacing;
+          height: 1px;
+          background-color: @oui-tile-item-border-color;
+        }
+      }
+
+      // stylelint-disable max-nesting-depth
+      &_button {
+        &:hover,
+        &:focus {
+          background-color: @oui-tile-background-hover;
```

Comment:
You should define the color property too, else, the ratio between background-color / color can be 	
insufficient.